src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -103,6 +103,7 @@ private Action<IModuleSymbol> Translate(Action<ModuleSymbol> action)
             Action<ModuleSymbol> symbolValidator = null,
             SignatureDescription[] expectedSignatures = null,
             string expectedOutput = null,
+            string[] args = null,
```

Comment:
>string[] args = null [](start = 12, length = 20)

It is not going to be commonly used, consider adding it to the end. Same for other similar places.